### ISSUE TITLE
Errors with the latest version

### ISSUE THREAD

**Issue by PARADOX070** [Hekili/hekili#225]
As of the latest version for 8.2.5 getting spammed with:

```
13x Hekili\Classes\DeathKnightBlood.lua:50: attempt to perform arithmetic on local 'start' (a nil value)
Hekili\Classes\DeathKnightBlood.lua:50: in function `reset'
Hekili\State.lua:5094: in function `reset'
Hekili\Core.lua:1097: in function `ProcessHooks'
Hekili\UI.lua:804: in function <Hekili\UI.lua:668>

Locals:
t = <table> {
 spend = <function> defined @Hekili\Classes\DeathKnightBlood.lua:70
 forecast = <table> {
 }
 regenModel = <table> {
 }
 gain = <function> defined @Hekili\Classes\DeathKnightBlood.lua:59
 last_tick = 0
 max = 0
 tick_rate = 0.100000
 regen = 0
 expiry = <table> {
 }
 timeTo = <function> defined @Hekili\Classes.lua:138
 modmax = 0
 fcount = 1
 times = <table> {
 }
 values = <table> {
 }
 actual = 0
 cooldown = 8.239428
 reset = <function> defined @Hekili\Classes\DeathKnightBlood.lua:45
}
(for index) = 1
(for limit) = 6
(for step) = 1
i = 1
start = nil
duration = nil
ready = nil
(*temporary) = <table> {
 1 = 0
 2 = 0
 3 = 0
 4 = 0
 5 = 0
 6 = 0
}
(*temporary) = nil
(*temporary) = "attempt to perform arithmetic on local 'start' (a nil value)"
state = <table> {
 spend = <function> defined @Hekili\State.lua:1255
 tostring = <function> defined =[C]:-1
 summonPet = <function> defined @Hekili\State.lua:904
 role = <table> {
 }
 IsKnown = <function> defined @Hekili\State.lua:5482
 forecastResources = <function> defined @Hekili\State.lua:1075
 target = <table> {
 }
 cast_start = 0
 GetTime = <function> defined =[C]:-1
 safenum = <function> defined @Hekili\State.lua:488
 IsCasting = <function> defined @Hekili\State.lua:4742
 UnitCastingInfo = <function> defined =[C]:-1
 essence = <table> {
 }
 tonumber = <function> defined =[C]:-1
 items = <table> {
 }
 holds = <table> {
 }
 resetting = true
 prev = <table> {
 }
 display = "Primary"
 UnitPower = <function> defined =[C]:-1
 removeStack = <function> defined @Hekili\State.lua:801
 prev_off_gcd = <table> {
 }
 active_dot = <table> {
 }
 applyDebuff = <function> defined @Hekili\State.lua:818
 mainhand_speed = 2.966000
 IsUsableSpell = <function> defined =[C]:-1
 debuff = <table> {
 }
 PlayerBuffUp = <function> defined @Hekili\State.lua:4360
 channelSpell = <function> defined @Hekili\State.lua:961
 equipped = <table> {
 }
 gain = <function> defined @Hekili\State.lua:1242
 removeBuff = <function> defined @Hekili\State.lua:765
 TimeToReady = <function> defined @Hekili\State.lua:5723
 rangefilter = false
 reduceCooldown = <function> defined @Hekili\State.lua:619
 iteration = 0
 setStance = <function> defined @Hekili\State.lua:879
 history = <table> {
 }
 args = <table> {
 }
 predictionsOn = <table> {
 }
 AddToHistory = <function> defined @Hekili\Events.lua:690
 ProcessEvent = <function> defined @Hekili\State.lua:4514
 buff = <table> {
 }
 SetupCycle = <function> defined @Hekili\State.lua:637
 table_insert = <function> defined =[C]:-1
 print = <function> defined @FrameXML\RestrictedInfrastructure.lua:116
 encounterName = "None"
 GetSpellCount = <function> defined =[C]:-1
 ClashOffset = <function> defined @Hekili\State.lua:5877
 azerite = <table> {
 }
 index = 0
 spell_targets = <table> {
 }
 offset = 0
 class = <table> {
 }
 runes = <table> {
 }
 putTrinketsOnCD = <function> defined @Hekili\State.lua:4381
 UnitPowerMax = <function> defined =[C]:-1
 QueuedCastRemains = <function> defined @Hekili\State.lua:4751
 stat = <table> {
 }
 ptr = 1
 pet = <table> {
 }
 ipairs = <function> defined =[C]:-1
 purge = <table> {
 }
 TargetBuffUp = <function> defined @Hekili\State.lua:4370
 pairs = <function> defined =[C]:-1
 pvptalent = <table> {
 }
 rune = <table> {
 }
 rawGain = <function> defined @Hekili\State.lua:1249
 GetItemGem = <function> defined =[C]:-1
 UnitIsFriend = <function> defined =[C]:-1
 selectionTime = 60
 ClearCycle = <function> defined @Hekili\State.lua:686
 combat = 0
 delayMax = 15
 false_start = 0
 addStack = <function> defined @Hekili\State.lua:780
 HasRequiredResources = <f
```

**Comment by Hekili** [comment]
Please submit a complete ticket.  I'm not able to replicate this with just the lua error.

**Comment by PARADOX070** [comment]
What else do you need? Everything disabled with the exception of hekili. Use a portal, Change zone's or phase over and you get greeted with these wow errors.

**Comment by PARADOX070** [comment]
And it only seems to happen on a DK.

**Comment by Hekili** [comment]
The instructions were there when you made the ticket.

**Comment by PARADOX070** [comment]
**Describe the Bug/Issue**
LUA Error

**To Reproduce**
#1 Log a DK.
#2 Disable all addons with the exception of Hekili.
#3 Take the portal from The Great Seal to Orgrimmar.
#4 Get greeted by error spam.

**Expected behavior**
Not being greeted by errors.

**Screenshots**
https://i.imgur.com/TVdVCcB.png
https://i.imgur.com/5QLNQPr.png

**PUT ISSUE REPORT LINK HERE:**
https://pastebin.com/usEU8KqQ
 
**PUT SNAPSHOT LINK HERE:**
There is nothing wrong with recommendations, Snapshot not needed.

**PUT ERROR MESSAGE(S) HERE:**
https://pastebin.com/qcDnGGcb

**Comment by PARADOX070** [comment]
Need anything else?

**Comment by Hekili** [comment]
Nope, should be addressed in 8.2.5-01.01, but let me know if you experience anything odd.
